pennylane_qiskit/converter.py

```diff
@@ -224,16 +225,43 @@ def _function(params: dict = None, wires: list = None):
                 gate = dagger_map[instruction_name]
                 qml.adjoint(gate)(wires=operation_wires)
 
+            elif isinstance(op, Measure):
+                meas_terminal = True
+                # Look-ahead for more gate(s) on its wire(s)
+                for meas_ix in range(idx, num_inst - 1):
```

Comment:
Just personal preference but I think this would be easier to read if we define a couple of lists, i.e.

```
instructions =[q.operation for q in qc.data]
qargs = [q.qubits for q in qc.data]
                
remaining_wires = [getattr(op, "base_class", op.__class__).__name__ for op in instructions[idx+1:]]
remaining_names = [set([wire_map[hash(q)] for q in qubits]) for qubits in qargs[idx+1]]
```

and then have the loop be over `name, wires in zip(remaining_names, remaining_wires)`. We could get rid of the `num_inst` variable, and the text below this would be much easier to read with just `name` and `wires` as variables.

I kind of want to re-organize the entire function to be easier to read though, so 🤷‍♀️

Reply:
I have made it a bit easier to read and removed the `num_inst` variable. 
However, I prefer the current method instead of looping over individual lists, as the latter would make us re-iterate Qiskit's quantum circuit at least three times. This would add a computational cost if the circuit would be huge.